library/core/src/marker.rs

```diff
@@ -315,6 +315,18 @@ pub trait StructuralEq {
 /// the trait `Copy` may not be implemented for this type; field `points` does not implement `Copy`
 /// ```
 ///
+/// Shared references (`&T`) are also `Copy`, so a type can be `Copy`, even when it holds
+/// shared references of types `T` that are *not* `Copy`. Consider the following struct,
+/// which can implement `Copy`, because it only holds a *shared reference* to our non-`Copy`
+/// type `PointList` from above:
```

Comment:
The documentation usually separates code block with an empty line above and below for clarity.

```suggestion
/// type `PointList` from above:
///
```

Reply:
Good catch! Thanks.